### PR TITLE
Test banner for warning of using istio < 1.10 with k8s 1.22

### PR DESCRIPTION
**Pull request by ericvn**
A PR to get capture discussion on publicizing the fact that Istio 1.9 and earlier do not work with Kubernetes 1.22 which was just released started in https://github.com/istio/istio.io/issues/10151.

Fixes https://github.com/istio/istio.io/issues/10154

### CODE REVIEW

**Comment by ericvn** [comment]
I will take any comments/changes to the message. Toning it done some:
```
Istio versions prior to 1.10 do not work with Kubernetes 1.22 or later. Verify your Istio version is supported when upgrading your Kubernetes version.
```

File: content/en/events/banners/kubernetes-1.22-with-older-releases.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+title: Kubernetes 1.22 Release statement
+period_start: 2021-08-04
+period_duration: 180
+max_impressions: 20
+timeout: 120
+link: /docs/releases/supported-releases#support-status-of-istio-releases
+
+---
+
+Warning! Kubernetes 1.22 is not supported by Istio release 1.9.x. Do not upgrade your Kubernetes version if you are using Istio versions prior to 1.10.
```
Comment (brian-avery) [inline content/en/events/banners/kubernetes-1.22-with-older-releases.md:11]: Is this better? 

```suggestion
Istio release 1.9.x is not compatible with Kubernetes 1.22. If you upgrade your Kubernetes cluster, please consider upgrading to Istio 1.10 or newer. 
```

Comment (ericvn) [reply content/en/events/banners/kubernetes-1.22-with-older-releases.md:11]: I'm not sure `consider` is accurate, at least if they are upgrading to 1.22.

**Comment by ericvn** [comment]
I suspect, depending on when this merges, it makes sense to CP to `istio.io`

**Comment by craigbox** [comment]
How about something.nice and positive like "Kubernetes 1.22 is supported by
Istio 1.10 and above. Removal of deprecated APIs (link) will  break older
versions of Istio. Check here for the supported version table"

**Comment by brian-avery** [comment]
I like it @craigbox

**Comment by brian-avery** [comment]
Is it worth it to clarify that by older versions, we mean 1.9 and below? Call them out explicitly?

**Comment by craigbox** [comment]
Something went wrong here...

image

**Comment by ericvn** [comment]
I wonder if it's a length limitation. I've shortened it and it seems to work.
Screen Shot 2021-08-06 at 8 23 13 AM

Now to see if the wording should be tweaked some more.

**Comment by craigbox** [comment]
Still failing for me, though my screen is _super wide_: but funnily enough, it fixes after a few refreshes.  Looking to see if I can find out why...

image

**Comment by ericvn** [comment]
I'm seeing the same when using the `Deploy` here, and seemed to correct itself on the 3rd refresh. Worked fine locally with `make serve`. Thanks for looking into it.

File: content/en/events/banners/kubernetes-1.22-with-older-releases.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: Kubernetes 1.22 Release statement
+period_start: 2021-08-04
+period_duration: 180
+max_impressions: 8
+timeout: 120
+link: /docs/releases/supported-releases#support-status-of-istio-releases
+---
+[Removal of deprecated APIs](https://kubernetes.io/blog/2021/07/14/upcoming-changes-in-kubernetes-1-22/) in
+Kubernetes 1.22 will break Istio 1.9 and below. Click here for the supported version table
```
Comment (craigbox) [inline content/en/events/banners/kubernetes-1.22-with-older-releases.md:10]:
```suggestion
Kubernetes 1.22 will only work with Istio 1.10 and above. Check here for the supported version table.
```

**Comment by ericvn** [comment]
At this point, the `deploy` seems to be OK (although may have to use a private window to get the banner to show). The link still takes you to the istio.io `support releases` page, but there is a PR to update that and it should be active after the release next week.

**Review by craigbox — APPROVED** [review]

**Pull request merged**